README.rst

```diff
@@ -103,13 +109,31 @@ Run the application against your local Cassandra:
 Testing the Endpoints
 =====================
 
-Requesting a new JWT (using the example credentials inserted into Cassandra above):
+Requesting a new JWT via Resource Owner Password Grant (using the example credentials inserted into Cassandra above):
 
 .. code-block:: bash
 
     $ curl --silent -X POST -u test0:test0 -d "grant_type=password&username=test0&password=test0&scope=uid" \
          "http://localhost:8080/oauth2/access_token?realm=/services" | jq .
 
+Requesting a new JWT via Implicit Flow:
```

Comment:
We should probably describe how to open the URL in the browser (and the consent screen step). The cURL commands are fine, but that's not what the flow usually looks like for humans :smirk: